src/Templating/src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-CSharp/Program.cs

```diff
@@ -12,9 +12,9 @@ namespace Company.WebApplication1
 {
     public class Program
     {
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
-            CreateWebHostBuilder(args).Build().Run();
+            await CreateWebHostBuilder(args).Build().RunAsync();
```

Comment:
Y NO `return`? :thinking:

Reply:
For templates establish the pattern before the optimization?

Straight return `Task rather` than `await Task` has caveats; like don't do it in a `using`, try/catch/finally etc

Reply:
Yes, awaiting is fine here.